tests/benchmarks/multicore/idle_gpio/testcase.yaml

```diff
@@ -0,0 +1,33 @@
+common:
+  sysbuild: true
+  tags: ci_build ci_tests_benchmarks_multicore
+  harness_config:
+    type: multi_line
+    ordered: true
+    regex:
+      - "Multicore idle test on"
+      - "Multicore idle test iteration 0"
+      - "Multicore idle test iteration 1"
+
+tests:
+  benchmarks.multicore.idle_gpio.nrf54h20dk_cpuapp_cpurad.s2ram:
+    platform_allow:
+      - nrf54h20dk/nrf54h20/cpuapp
+    integration_platforms:
+      - nrf54h20dk/nrf54h20/cpuapp
+    extra_args:
+      SB_CONF_FILE=sysbuild/nrf54h20dk_nrf54h20_cpurad.conf
+      CONFIG_PM=y CONFIG_PM_S2RAM=y CONFIG_POWEROFF=y CONFIG_PM_S2RAM_CUSTOM_MARKING=y
+      CONFIG_CONSOLE=n CONFIG_UART_CONSOLE=n CONFIG_SERIAL=n CONFIG_GPIO=y CONFIG_BOOT_BANNER=n
+      CONFIG_NRFS_MRAM_SERVICE_ENABLED=n CONFIG_CLOCK_CONTROL=n
+      remote_CONFIG_PM=y remote_CONFIG_POWEROFF=y remote_CONFIG_CONSOLE=n
+      remote_CONFIG_UART_CONSOLE=n remote_CONFIG_SERIAL=n remote_CONFIG_GPIO=y
+      remote_CONFIG_BOOT_BANNER=n remote_CONFIG_NRFS_MRAM_SERVICE_ENABLED=n
+      remote_CONFIG_CLOCK_CONTROL=n
+      DTC_OVERLAY_FILE="boards/nrf54h20dk_nrf54h20_cpuapp_s2ram.overlay"
+      remote_DTC_OVERLAY_FILE="boards/nrf54h20dk_nrf54h20_cpurad_s2ram.overlay"
+    harness: pytest
+    harness_config:
+      fixture: ppk_power_measure
+      pytest_root:
+        - "${CUSTOM_ROOT_TEST_DIR}/test_measure_power_consumption.py::test_measure_and_data_dump_power_consumption"
```

Comment:
this will not work for gpio wakeup, it is prepared only for periodic wakeups.
For now that part needs to be skipped.